spec/classes/init_spec.rb

```diff
@@ -3241,4 +3241,16 @@
       end
     end
   end
+
+  describe 'with parameter sshd_addressfamily' do
+    let(:facts) do
```

Comment:
I would add something like this to test all valid values that result in the entry being in the config

``` ruby
['any','ipv4','ipv6'].each do |value|
  context "set to a valid entry of #{value}" do
     .. your code here
  end
end
```

Another context should set it to undef and show `.without_content(/^\s*AddressFamily/)`

And another context that shows if you specify an invalid value that you get an error.

This way we test that valid entries get the right config. Undef is still valid, though it ensures that the line is not present, and invalid entries cause an error.